runtime/near-vm-logic/tests/test_context.rs

```diff
@@ -11,7 +11,7 @@ pub fn create_context() -> VMContext {
         signer_account_pk: vec![0, 1, 2, 3, 4],
         predecessor_account_id: "carol".to_owned(),
         input: vec![0, 1, 2, 3, 5],
-        block_index: 10,
+        height: 10,
```

Comment:
Word "block" should not be omitted. Block height is more descriptive. We are not omitting word block in block timestamp, and we are not omitting "attached" in the "attached deposit" either.

Reply:
I checked manually and found that for many cases using `block_height` instead `height` is useless. We don't use `block_height_created` and `block_height_included`, we use `height_created` and `height_included`. We use `height` in `BlockHeaderInnerLite`. We use `get_block_hash_by_height()` instead of `get_block_hash_by_block_height()`.

If you still want to add `block_` to `height` somewhere, please point all the places where you think it's necessary.

Reply:
That's because chain and runtime have different contexts. When some developer is working with chain and sees "height" they obviously know that this height is related to block height. The runtime does not have this context, because it is an engine that executes stuff (it does not need blockchain to run) so when someone encounters "height" inside the runtime code they don't know what it relates to. I suggest we use "block_height" everywhere in the `nearcore/runtime` and `nearcore/core` (unless the primitives like `block.rs`). The reason why we would keep "block" in `nearcore/core` is because it does not have context of the "chain" either.

Reply:
According to your comment, I've updated some "height"s to "block_height"s where it makes sense.